cdk/bin/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const AWS_REGION = 'eu-west-1';
+export const STACK = 'media-service';
+export const REAPER_APP_NAME = 'grid-reaper';
+export const ENV_VAR_BATCH_SIZE = 'BATCH_SIZE';
```

Comment:
this is just a tiny bit confusing because I am assuming the variable name is ENV_VAR_BATCH_SIZE and the value is a number (string but number) but it sounds like the variable name is "BATCH_SIZE"? I could be reading wrong. But maybe clearer if we just use the string in place?

Reply:
this is a variable representing the name of the environment variable, and it's in the `constants.ts` so it can be referred to when it's **set** in the CDK stack, and then when its **read** in the lambda code. Having a constant for this avoids there ever being a discrepancy when using that value.

Reply:
ok, I think I have a mild preference for using the string in place but I will leave it up to you, it's really not important :)